README.md

```diff
@@ -4,33 +4,40 @@ Open source voice assistant development. Make your own minds!
 
 Open Assistant is a private open source personal assistant prototype able to complete operating system tasks using vocal commands.
 
-# Installation
+## Installation
 
-## Windows
+### Windows
 
-Windows (Click & Run)
+#### _Click & Run_
 
-* Download http://openassistant.org/download/oa_0.21_windows.zip
+* Download [http://openassistant.org/download/oa_0.21_windows.zip](http://openassistant.org/download/oa_0.21_windows.zip)
 * Unzip package
 * Launch `oa.exe` in "Open Assistant" folder
 
-Windows Python Install (Python 3.x)
+#### _Python 3_
 
 * Install [common list of Python packages](https://www.python.org/downloads/windows/)
 * Install Windows-specific dependencies: ``pip install pywin32``
 * Install Python dependencies: `pip install -r requirements.txt`
 
-## Ubuntu Linux
+### Mac OS X
+
+* Install [brew](https://brew.sh/) if you have not already.
+* Install pocketsphinx dependencies: `brew install swig`, `brew install espeak`, and `brew install openal-soft`
+* Link the openal-soft libraries to the include path: `ln -s /usr/local/Cellar/openal-soft/1.20.1/include/AL/* /usr/local/include`
```

Comment:
This is just awful.. it'll break when/if the `openal-soft` version changes.. it's weird.. even worse, it seems to work very well.

Reply:
I hear you. Unfortunately, that's the way that the brew installation method for openal-soft works. 

The brew installation instructions even tell you that you need to link it after the fact. I figured installation using a common package manager is more preferable over alternative methods for this route.

Reply:
```shell
openal-soft: stable 1.20.1 (bottled), HEAD [keg-only]
Implementation of the OpenAL 3D audio API
https://openal-soft.org/
/usr/local/Cellar/openal-soft/1.20.1 (28 files, 1.4MB)
  Poured from bottle on 2020-08-26 at 15:15:47
From: https://github.com/Homebrew/homebrew-core/blob/HEAD/Formula/openal-soft.rb
License: LGPL-2.0
==> Dependencies
Build: cmake ✘, pkg-config ✘
==> Options
--HEAD
	Install HEAD version
==> Caveats
openal-soft is keg-only, which means it was not symlinked into /usr/local,
because macOS provides OpenAL.framework.

If you need to have openal-soft first in your PATH run:
  echo 'export PATH="/usr/local/opt/openal-soft/bin:$PATH"' >> ~/.zshrc

For compilers to find openal-soft you may need to set:
  export LDFLAGS="-L/usr/local/opt/openal-soft/lib"
  export CPPFLAGS="-I/usr/local/opt/openal-soft/include"

==> Analytics
install: 294 (30 days), 1,019 (90 days), 4,601 (365 days)
install-on-request: 279 (30 days), 965 (90 days), 4,355 (365 days)
build-error: 0 (30 days)
```

Reply:
I suppose we could instruct them to add it to their path as the altnerative to soft linking.

Reply:
Until I have a chance to test it out, I'd say lets go with the symbolic link for now, and I'll make a note to test it out and update if applicable.

Reply:
Pocketsphinx has been one of the trickier dependencies, and it's tended to change over time. Agree with your assessment. Exporting a variable seems a little better (if that works), but the options are pretty limited.